### PR TITLE
Test connection in action test

### PR DESCRIPTION
**Pull request by sfc-gh-turbaszek**
### Pre-review checklist
   * [ ] I've confirmed that instructions included in README.md are still correct after my changes in the codebase.
   * [ ] I've added or updated automated unit tests to verify correctness of my new code.
   * [ ] I've added or updated integration tests to verify correctness of my new code.
   * [ ] I've confirmed that my changes are working by executing CLI's commands manually.
   * [ ] I've confirmed that my changes are up-to-date with the target branch.
   * [ ] I've described my changes in the release notes.
   * [ ] I've described my changes in the section below.

### Changes description
...

### CODE REVIEW

File: .github/workflows/test_cli_action.yaml [hunk 1]
```diff
@@ -3,15 +3,23 @@ name: "CLI Action testing"
 on:
   schedule:
     - cron: "0 0 * * *"
+  workflow_dispatch:
```
Comment (sfc-gh-astus) [inline .github/workflows/test_cli_action.yaml:6]: Empty workflow_dispatch?

File: .github/workflows/test_cli_action.yaml [hunk 2]
```diff
@@ -3,15 +3,23 @@ name: "CLI Action testing"
...
 
 jobs:
   version:
     name: "Check Snowflake CLI version"
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
```
Comment (sfc-gh-astus) [inline .github/workflows/test_cli_action.yaml:14]: - uses: actions/checkout@v4
        with:
          persist-credentials: false

File: .github/workflows/test_cli_action.yaml (hunk 1 above)
Comment (sfc-gh-turbaszek) [reply .github/workflows/test_cli_action.yaml:6]: Takes no input, allows triggering the workflow manually

File: .github/workflows/test_cli_action.yaml (hunk 2 above)
Comment (sfc-gh-turbaszek) [reply .github/workflows/test_cli_action.yaml:14]: Fixed 👌

**Review by sfc-gh-astus — APPROVED** [review]

**Pull request merged**